cmd/util.go

```diff
@@ -237,3 +238,11 @@ func ifNotBlank(s string, defaultValue string) string {
 	}
 	return s
 }
+
+func toAbsPath(p string, config *string, defaultParentPath ...string) path.ResourcePath {
```

Comment:
would it make sense to have `realXPath` call this, since it seems like you're calling this immediately before calling any of those everywhere?

Reply:
Thanks for the suggestion!